src/tests/nativeaot/SmokeTests/Preinitialization/Preinitialization.csproj

```diff
@@ -3,7 +3,8 @@
     <OutputType>Exe</OutputType>
     <CLRTestPriority>0</CLRTestPriority>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <CLRTestTargetUnsupported Condition="'$(IlcMultiModule)' == 'true'">true</CLRTestTargetUnsupported>
+    <!-- TODO-LLVM: Fails on Wasm: Assert.AreEqual(unchecked((byte)GetFloat()), FloatConversions.s_byte); -->
+    <CLRTestTargetUnsupported Condition="'$(IlcMultiModule)' == 'true' or '$(TargetsWasm)' == 'true'">true</CLRTestTargetUnsupported>
```

Comment:
Could you exclude it in `dirs.proj` instead and open an issue about this?

Reply:
Yes, issue created.  https://github.com/dotnet/runtimelab/issues/3074